### ISSUE TITLE
Add documentation on how to add new events for preview

### ISSUE THREAD

**Issue by shravan20** [shravan20/fosstrak-india#11]
# FOSSTrak Feature Request: Document Contribution Process for Adding Events

```
TLDR;

This feature request proposes adding documentation to the FOSSTrak project detailing 
the process for contributing events to the repository. The documentation will include 
step-by-step instructions on forking the repository, cloning it, adding event information 
in JSON format, committing changes, and creating a pull request (PR). This documentation 
will enhance accessibility for contributors and foster community engagement. 

Thank you for considering this enhancement to the project documentation!
```

### Feature Description

Add documentation detailing the contribution process for adding events to the repository using a specific JSON format.

### Use Case

Users who want to contribute events to the repository need clear instructions on how to format and submit event information.

### Proposed Solution

Create a new section in the project documentation outlining the step-by-step process for contributing events:
- Forking the repository
- Cloning the forked repository
- Adding event information in JSON format
- Committing changes and creating a pull request (PR)

### Alternatives Considered

We considered providing a video tutorial or conducting live workshops, but a written guide in the documentation is more accessible and scalable.

### Dependencies

The documentation update depends on having a clear understanding of the JSON format required for event submissions.

Here is the sample format for reference:

```
{
    "name": "Test Event 2024",
    "shortName": "TE'24",
    "shortDescription": "This is a simple One liner Description Test event",
    "description": "Description Test event",
    "imageUrl": "https://res.cloudinary.com/dwzmsvp7f/image/fetch/q_75,f_auto,w_400/https%3A%2F%2Fmedia.insider.in%2Fimage%2Fupload%2Fc_crop%2Cg_custom%2Fv1711572509%2Fkgg1nz7mkuaq8tltncgo.png",
    "startDate": null,
    "endDate": null,
    "website": "www.ohmyscript.com",
    "location": {
      "title": "Saibeen Complex, Mangalore",
      "lat": "12.26",
      "lng": "122.24"
    },
    "type": "Free",
    "presentationForm": "ONLINE",
    "registrationLink": "http://ohmyscript.com/registration",
    "mainOrganisers": [],
    "hashTags": [],
    "bannerLink": [
      {
        "link": "",
        "isActive": true,
        "isMain": true
      }
    ],
    "socialAccounts": [
      {
        "type": "MASTODON",
        "identified": "https://mastodon.social/@shravan20"
      }
    ]
  }
```
